app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,41 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>PrivateEvents</title>
+    <title>Private Events</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
-
   <body>
-    <%= yield %>
+    <div class="container-md">
+      <nav class="navbar navbar-expand-lg navbar-light bg-light flex justify-content-between">
+        <%= link_to 'Private Events', root_path, class: "navbar-brand font-weight-bolder" %>
+
+        <div class="d-flex flex-row align-items-center">
+          <% if current_user %>
+            <%= link_to 'New Event', new_event_path, class: 'btn btn-success mr-2' %>
+
+            <%= link_to current_user.full_name, user_path(current_user), id: "user_#{current_user.id}", class: "pr-2" %>
+
+            <%= link_to 'Sign Out', destroy_user_session_path, :method => :delete %>
+          <% elsif !current_user && controller_name == 'sessions' %>
+            <%= link_to "Sign up", new_user_registration_path %><br />
+          <% else %>
+            <%= link_to "Log in", user_session_path %>
+          <% end %>
```

Comment:
- [ ] Since this method has more than one line of markup in the condition, create a method that renders a partial that contains all the markup within the if statement.